app/views/suppliers/show.html.erb

```diff
@@ -1,63 +1,70 @@
-<div id="supplier_home">
-	<h1><%= @supplier.name %> Supplier Page</h1>
-	<div class="open-orders">
-		<h2>Open Orders</h2>
-		<%= link_to "View All Orders", supplier_orders_path %><br>
-		<%= link_to "Manage Items", supplier_items_path %>
-		<div class="panel-group" id="accordion">
-      <div class="panel panel-default">
-        <div class="panel-heading">
-        	<h4 class="panel-title" >
-						<a data-toggle="collapse" data-parent="#accordion" href="#collapseOne"><%= @supplier.name  %> Account Details</a>
-          </h4>
-        </div>
-        <div id="collapseOne" class="panel-collapse account-details collapse ">
-          <div class="panel-body">
-						<p><%= @supplier.name %></p>
-						<p><%= @supplier.description %></p>
-						<div class="create text-right">
-							<button type="button" class="btn btn-lg btn-default create" data-toggle="modal" data-target=".reg-modal">Edit</button>
-							<div class="modal fade reg-modal" tabindex="-1" role="dialog" aria-labelledby="mySmallModalLabel" aria-hidden="true">
-								<div class="modal-dialog modal-lg">
-									<div class="modal-content">
-										<%= render "partials/supplier_edit_info_form"	%>
-									</div>
-								</div>
-							</div>
-						</div>
+<div id="supplier_home" class="row col-md-12">
+  <div class="row col-md-1">
+  </div>
+  <div class="row col-md-10">
+    <h1><%= @supplier.name %></h1>
+    <div class="open-orders">
+      <div id="supplier-links">
+        <%= link_to "View All Orders", supplier_orders_path %><br>
+        <%= link_to "Manage Items", supplier_items_path %>
```

Comment:
I'd like to see these two links have the same style as the two panel-headings below. I think if we just wrap them in a class with that, that should be all we need. Let's work on it.